### ISSUE TITLE
[Bug Report] Jumbotron image gets pushed down by expanding content

### ISSUE THREAD

**Issue by nolde** [vuetifyjs/vuetify#3786]
### Versions and Environment
**Vuetify:** 1.0.13
**Vue:** 2.5.16
**Browsers:** Chrome 65.0.3325.181
**OS:** Windows 10

### Steps to reproduce
1. Add a Jumbotron with an image (using `src`)
2. Add content to it that expands (in my example, I use a stepper)
3. Change children height by activation expansion

### Expected Behavior
Jumbotron image is still attached to top.

### Actual Behavior
Image gets pushed down, looking weird.

### Reproduction Link
[https://codepen.io/anon/pen/EEdQdO](https://codepen.io/anon/pen/EEdQdO)

**Comment by johnleider** [comment]
Temporary workaround: https://codepen.io/johnjleider/pen/QmzoXQ

**Comment by nolde** [comment]
Thanks for the workaround.

In the meantime, I actually dropped Vuetify's Jumbotron and made one myself. This one is actually hard to fix, and I would say that it may not be a Vuetify problem, but just the way browsers handle expanding content. 

I was able to fix it using my own component, but I had to forgo the gradient (I applied it over the image directly, saved it, and used the new image). It works for now, so I am happy. If you find a way to make this work, it would be nice, but I would not stress too much over it.

Thanks for taking the time to look into it.
